tests/worker/test_all_workers.py

```diff
@@ -21,3 +30,66 @@ async def test_should_get_manifest(pool: WorkerPool) -> None:
     async with pool.get_worker(PACKAGE, 1, 1) as worker:
         manifest = await worker.get_manifest()
         assert manifest == PACKAGE.manifest
+
+
+# TODO: Include a test package with actual static files instead of whatever this is.
+
+
+def _inject_static_file_into_dist(package: DirPackageLocation, name: str, content: str) -> None:
+    full_path = package.path / f"{DIST_DIR}/{name}"
+    full_path.parent.mkdir(exist_ok=True)
+    full_path.write_text(content)
+
+
+def _inject_static_file_into_manifest(package: DirPackageLocation, name: str, size: int) -> None:
+    package.manifest = deepcopy(package.manifest)
+    package.manifest.static_files[name] = PackageFile(mime_type=mimetypes.guess_type(name)[0], size=size)
+    (package.path / DIST_DIR / MANIFEST_FILENAME).write_text(package.manifest.model_dump_json())
+
+
+_STATIC_FILE_NAME = "static/test_file.txt"
+_STATIC_FILE_CONTENT = "static example file\n"
+
+
+async def test_should_get_static_file_from_dir(pool: WorkerPool) -> None:
+    with PACKAGE.as_dir_package() as package:
+        _inject_static_file_into_dist(package, _STATIC_FILE_NAME, _STATIC_FILE_CONTENT)
+        _inject_static_file_into_manifest(package, _STATIC_FILE_NAME, len(_STATIC_FILE_CONTENT))
+
+        worker: Worker
+        async with pool.get_worker(package, 1, 1) as worker:
+            static_file = await worker.get_static_file(_STATIC_FILE_NAME)
+            assert static_file.data == _STATIC_FILE_CONTENT.encode()
+            assert static_file.mime_type == "text/plain"
+            assert static_file.size == len(_STATIC_FILE_CONTENT)
+
+
+async def test_should_raise_FileNotFoundError_when_not_in_manifest(pool: WorkerPool) -> None:
```

Comment:
Warum nicht einfach snake case?

`test_should_raise_file_not_found_error_when_not_in_manifest`

Reply:
Weil es so einfacher lesbar ist. Die Namen von Test-Funktionen sollten mMn immer so nah wie möglich an einem ordentlichen Satz sein. Snake case hat hier einfach keinen Vorteil.

Reply:
Widerstrebt es dir den case von `FileNotFoundError` zu verändern, weil es für dich lesbarer erscheint als `file_not_found_error`?

Falls ich dich richtig verstehe,  wäre eine Möglichkeit stattdessen einfach `test_should_raise_when_not_in_manifest` zu schreiben.

In jedem Fall, denke ich nicht, dass der Fall rechtfertigt die Python-Naming-Konvention zu verletzen, nämlich für Funktionen snake case zu verwenden.

Reply:
> Widerstrebt es dir den case von FileNotFoundError zu verändern, weil es für dich lesbarer erscheint als file_not_found_error?

Ja, die Exception heißt halt `FileNotFoundError`.

Ich bin es gewohnt, dass bei Testfunktionen andere (zumeist _weniger_) Naming Conventions benutzt werden als bei "normalen" Funktionen. (Klassiker aus Java und Kotlin: [1], [2])  Mir ist das am Ende aber auch nicht wichtig genug, um lange drüber zu diskutieren, ich passe mir euch/dir an.

[1]: https://dzone.com/articles/7-popular-unit-test-naming
[2]: https://kotlinlang.org/docs/coding-conventions.html#names-for-test-methods